src/test/java/mate/academy/dao/impl/UserDaoImplTest.java

```diff
@@ -45,40 +49,40 @@ void setUp() {
     @Test
     void save_ok() {
         User actual = userDao.save(user);
-        Assertions.assertNotNull(actual);
-        Assertions.assertEquals(ID, actual.getId());
+        assertNotNull(actual);
+        assertEquals(ID, actual.getId());
     }
 
     @Test
     void findByEmail_ok() {
         userDao.save(user);
         Optional<User> actual = userDao.findByEmail(EMAIL);
-        Assertions.assertNotNull(actual.get());
-        Assertions.assertEquals(actual.get().getId(), ID);
-        Assertions.assertEquals(actual.get().getEmail(), EMAIL);
-        Assertions.assertEquals(actual.get().getPassword(), PASSWORD);
+        assertTrue(actual.isPresent());
+        assertEquals(actual.get().getId(), ID);
+        assertEquals(actual.get().getEmail(), EMAIL);
+        assertEquals(actual.get().getPassword(), PASSWORD);
     }
 
     @Test
     void findByEmail_notOk() {
         userDao.save(user);
-        Assertions.assertThrows(NoSuchElementException.class, () -> {
-            userDao.findByEmail(ERROR).get();
+        assertThrows(NoSuchElementException.class, () -> {
+            userDao.findByEmail(ERROR).orElseThrow();
         }, "NoSuchElementException expected");
     }
 
     @Test
     void findById_Ok() {
         userDao.save(user);
-        User actual = userDao.findById(ID).get();
-        Assertions.assertNotNull(actual);
+        User actual = userDao.findById(ID).orElseThrow();
```

Comment:
adding `assertTrue` on the `isPresent()` check would give us more informative messages than just `NoSuchElementException` in case of test failing